README.md

```diff
@@ -92,6 +92,14 @@ After the container is started, you will be prompted to login to your Azure acco
 
 This build includes preview support for connecting to private registries (such as those described in Docker Hub [documentation](https://docs.docker.com/registry/deploying/)).  At the moment, OAuth is not supported, only basic authentication.  We hope to extend this support in the future.
 
+## Self-signed and corporate certificates
+
+If you are using a self-signed or corporate CA certificate (e.g. for a private Docker registry) and have the certificate authority's certificate registered in the Windows or Mac certificate store, you will need to set the `docker.useCertificateStore` setting to true in order for that certificate to be trusted by the extension. If you're on Linux, you will need to specify your certificate folder(s) and/or file(s) using the `docker.certificatePaths` setting. The exact folder to use will depend on the Linux distribution, but we suggest trying `["/etc/ssl/certs/ca-certificates", "/etc/openssl/certs", "/etc/pki/tls/certs", "/usr/local/share/certs"]` first.
+
+Note that currently if you specify either of these settings, Node.js's internal certificate list will be ignored.
```

Comment:
I find one of these to be more readable: 

Note that Node.js's internal certificate list will be ignored if you specify either of these settings.

OR 

Note that if you currently specify either of these settings, Node.js's internal certificate list will be ignored.

Reply:
Wait 'til next PR...